forge/ee/lib/billing/index.js

```diff
@@ -68,6 +68,19 @@ module.exports.init = async function (app) {
                 cancel_url: `${app.config.base_url}/team/${team.slug}/overview`
             }
 
+            // Use existing Stripe customer
+            const existingLocalSubscription = await app.db.models.Subscription.byTeam(team.id)
+            if (existingLocalSubscription?.customer) {
+                sub.customer = existingLocalSubscription.customer
+
+                // Required for tax_id_collection
+                sub.customer_update = {
+                    name: 'auto'
+                }
+            } else if (user?.email) {
+                sub.customer_email = user.email
```

Comment:
Passing along the users email is convenient, but I note that Stripe disables the email field entirely, meaning they can't edit it. I can see _potential_ examples were they might want to change it. e.g. to billing@company.com or team2@example.com; should I roll back pre-populating it @knolleary @knolleary @ZJvandeWeg?

<img width="988" alt="Screenshot 2022-12-16 at 17 45 47" src="https://user-images.githubusercontent.com/507155/208136136-6a739ed6-4675-4a79-8c23-de122e8c8021.png">

Reply:
I'd vote for letting them change it, if you don't pass it does Stripe re-use the one already linked to the subscription?

Reply:
If you're renewing your subscription, there's never an option on the checkout to change it.

If you're creating your subscription for the first time and we pass along an email, it also doesn't let the user change it.

Reply:
OK, I'd still vote for letting the user set it, because the team owner may not be the accounts department who should get the bill

Reply:
[75ead38](https://github.com/flowforge/flowforge/pull/1456/commits/75ead388e51b841ec4b568f80b965afbf62870d7)